Clarify any_errors_fatal behavior change in 2.17 porting guide (#1815)
ansible/ansible-documentation: docs/docsite/rst/porting_guides/porting_guide_core_2.17.rst

       assert:
         that: '"hi mom" is in untrusted_result.stdout'

> * ``any_errors_fatal`` - when a task fails on a host in a block with a ``rescue``
>   section the ``rescue`` section is executed on all hosts as all hosts are failed
>   automatically due to ``any_errors_fatal``.

Reviewer: ```suggestion
* ``any_errors_fatal`` - when a task in a block with a ``rescue`` section
  fails on a host, the ``rescue`` section is executed on all hosts. This
  occurs because ``any_errors_fatal`` automatically fails all hosts.
```

Author: Thanks!
